desks-web-app/src/components/booking-components/WeekOverviewRow.vue

```diff
@@ -20,7 +20,7 @@
   </v-row>
   <v-row>
     <v-col>
-      <v-window v-model="window">
+      <v-window v-model="window" :style="{marginTop: '-1em'}">
```

Comment:
https://vuetifyjs.com/en/components/slide-groups/

It guess a slide group would be more suitable to the kind of functionality you are building. Window made sense for the existing implementation because each window represented a specific workweek. Your suggestion is more a continues stream of the upcoming workdays.